packages/dialog/README.md

```diff
@@ -0,0 +1,82 @@
+# `@undercoatcss/dialog`
+
+## Installation
+
+```sh
+npm install @undercoatcss/dialog
+```
+
+## Initialization
+
+### Javascript
+```javascript
+import Dialog from '@undercoatcss/dialog';
+```
+
+### CSS
+```css
+@import '@undercoatcss/dialog/dialog';
+```
+
+## HTML layout
+
+```html
+<div class="dialog" id="{id}">
```

Comment:
I think it will be better to add `role="dialog"` for a11y

Reply:
Agree with @niktariy. 
But if we are using the `role=dialog` so we need a label for it. 
[Role Dialog MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role)

The quote from the article:
>However, adding role="dialog" alone is not sufficient to make a dialog accessible. 
>Additionally, the following needs to be done:
> 1. The dialog must be properly labeled
> 2. Keyboard focus must be managed correctly

Reply:
Okay, then what about `aria-modal="true"`?

Reply:
Okay, then what about `aria-modal="true"`?

Reply:
@glafirazhur, what does it mean "labeled"?
`
<button id="open-dialog-button"></button>
<div class="dialog" id="dialog" role="dialog" aria-labelledby="open-dialog-button"></div>
`

Reply:
@Sakalou it means that we have, for example, title, that can be used as label for modal window. So `aria-labelledby="dialog1Title"` refers to title:
```html
<div role="dialog" aria-labelledby="dialog1Title" aria-describedby="dialog1Desc">
  <h2 id="dialog1Title">Your personal details were successfully updated</h2>
  <p id="dialog1Desc">You can change your details at any time in the user account section.</p>
  <button>Close</button>
</div>
```

Reply:
@niktariy sometimes dialog can have no title and description, but in other cases we can add aria attributes

Reply:
@Sakalou yes, that’s why I propose to use `aria-modal=“true”` :)